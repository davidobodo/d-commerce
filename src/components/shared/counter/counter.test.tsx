import React from 'react';
import { cleanup, render } from '@testing-library/react';
import '@testing-library/jest-dom';

import Counter from './counter';

afterEach(cleanup);

const defaultProps = {
    initialcount: 1,
    maxValue: 10,
    setValue: jest.fn()
}

it('counter should match snapshot', () => {
    const { container } = render(<Counter {...defaultProps} />)
    expect(container).toMatchSnapshot();
})

