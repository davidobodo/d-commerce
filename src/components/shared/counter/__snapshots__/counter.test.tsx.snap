// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`counter should match snapshot 1`] = `
<div>
  <div
    class="sc-AxjAm eoxFJy"
  >
    <div
      class="counter"
    >
      1
    </div>
    <div
      class="editor"
    >
      <div
        class="increase btn"
        id="+"
      >
        +
      </div>
      <div
        class="decrease btn"
        id="-"
      >
        -
      </div>
    </div>
  </div>
</div>
`;
